### DISCUSSION TITLE
Using with an existing sorceress

### DISCUSSION THREAD

**Discussion by ghost** [isid0re/SoloLeveling#123] [Q&A]
I already have a Blizzballer that is in A1 Hell, but when I start the script it will go to the Den of Evil then just teleport around without attacking anything. Is there something I need to do before this can work with an existing character or does this only work with fresh toons? Thanks and please forgive my ignorance.

**Comment by isid0re** [comment]
Using it with an existing bot, you run the risk of losing equipped items if they dont match what the bot is looking for. 

Also if it's doing the den but not attacking anything double check you have installed everything. The attack settings are set by the config/builds/  autobuild files. If your blizzballer skills differ from what the bots looking for in the template. It wont work.

**Reply by isid0re** [reply]
Just found a problem with the leveling build where it mix and matched blizzballer and meteorb. That could cause your problem as well. Over updated the files. So try updating first.

**Reply by Borega** [reply]
Was about to write smth on that issue. Nice to see that it was not only on my side

**Reply by ghost** [reply]
The update appeared to work.. kinda. She is casting blizzard now, but in the Den she keeps trying to skip the zombies despite having fireball. She teleports around and spams blizzard and maybe throws out a single fireball. I'll mess around with the config a little bit to see if I can get it to match my Blizzballer

**Reply by isid0re** [reply]
I will level up a sorc with this build and see how it works in hell to compare notes with you. Thanks.

**Reply by ghost** [reply]
> 
> 
> I will level up a sorc with this build and see how it works in hell to compare notes with you. Thanks.

I also started a new sorc! She killed Baal in normal but sometime between last night and this morning stopped functioning correctly. She no longer attacks while pathing and has been getting stuck in packs and chickening out of the game.

Edit: I forced a respec and she's attacking now. Did she miss a trigger for the respec? Because she did Baal just fine using the Start build but once she hit 29-30 she stopped attacking while pathing and got stuck in a loop.
